backend/models/order.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+var address = {
+  addressName: { type: String },
+  name: { type: String },
+  addressLine: { type: String },
+  city: { type: String },
+  state: { type: String },
+  zipCode: { type: String },
+  phone: { type: String },
+};
+
+var orderSchema = new Schema(
+  {
+    orderID: { type: String },
```

Comment:
orderID, customerID and creditCard all should be ObjectID's as follows: `{ type: Schema.Types.ObjectID }`.

Reply:
Changed as requested.